src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetViewModels.cs

```diff
@@ -0,0 +1,113 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters.SystemJson;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using System.Text.Json.Serialization;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+/// <summary>
+/// Describes and provides high-level information about a data set.
+/// </summary>
+public record DataSetViewModel
+{
+    /// <summary>
+    /// The ID of the data set. 
+    /// </summary>
+    public required Guid Id { get; init; }
+
+    /// <summary>
+    /// The title of the data set.
+    /// </summary>
+    public required string Title { get; init; }
+
+    /// <summary>
+    /// A summary of the data set’s contents.
+    /// </summary>
+    public required string Summary { get; init; }
+
+    /// <summary>
+    /// The status of the data set. Can be one of the following:
+    ///
+    /// - `Published` - the data set has been published and will receive updates
+    /// - `Deprecated` - the data set is being discontinued and will no receive updates
+    /// - `Unpublished` - the data set has been unpublished and can no longer be used
+    /// </summary>
+    public required DataSetStatus Status { get; init; }
+
+    /// <summary>
+    /// The latest published data set version.
+    /// </summary>
+    public required DataSetLatestVersionViewModel LatestVersion { get; init; }
+
+    /// <summary>
+    /// The ID of the data set that supersedes this data set (if it has been deprecated).
+    /// </summary>
+    public Guid? SupersedingDataSetId { get; init; }
+}
+
+/// <summary>
+/// Describes and provides high-level information about the latest version for a given data-set.
```

Comment:
Similar to other comment, could drop the `Describes and` part. Also, it should be `data set`, not `data-set`.

Would go with `Provides high-level information the latest version of a data set.`